src/components/App.js

```diff
@@ -46,6 +48,9 @@ function App() {
 
               {/* Forum Pages */}
               <Route path="/create-forum" component={CreateForum} />
+              <Route path="/forums" component={Forums} />
+              <Route path="/forum/:forumID" component={ForumCard} />
```

Comment:
make these into `<PrivateRoute>` components to prevent users that are not logged in from accessing these pages